saleor/order/utils.py

```diff
@@ -307,7 +307,7 @@ def sum_order_totals(qs):
 
 def get_valid_shipping_methods_for_order(order: Order):
     return ShippingMethod.objects.applicable_shipping_methods_for_instance(
-        order, price=order.get_subtotal().gross
+        order, channel=order.channel, price=order.get_subtotal().gross
```

Comment:
Maby better to pass only `channel id` instead of getting channel object from the database? 

```suggestion
        order, channel_id=order.channel_id, price=order.get_subtotal().gross
```